fastlane/Fastfile

```diff
@@ -125,6 +125,12 @@ end
   desc "Updates a release branch for a new beta release"
   lane :new_beta_release do | options |
     ios_betabuild_prechecks(options)
+    ios_update_metadata(options)
+    violations = ios_lint_localizations(input_dir: 'Simplenote', abort_on_violations: false)
+    unless (violations.nil? || violations.empty?)
+      abort = UI.confirm("Violations found, we suggest fixing them in GlotPress first and retrying the beta afterwards. Abort?")
+      UI.abort_with_message!("Aborting the beta due to localization violations. Reach out to i18n folks to help you fix them.") if abort
+    end
```

Comment:
Nice!

Suggestion for future improvement @AliSoftware: move this into the action itself. I'm not sure what could be a good parameter name to enable this behavior 🤔 `interactive: true` maybe?